cfme/tests/v2v/test_migration_throttling.py

```diff
@@ -0,0 +1,89 @@
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+from cfme.v2v.migration_settings import MigrationSettings as migration_settings
+
+pytestmark = [
+    test_requirements.v2v,
+    pytest.mark.provider(
+        classes=[OpenStackProvider, RHEVMProvider],
+        selector=ONE_PER_VERSION,
+        required_flags=["v2v"],
+        scope="module"
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name="source_provider",
+        required_flags=["v2v"],
+        scope="module",
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup"),
+]
+
+
+@pytest.mark.parametrize(
+    "mapping_data_multiple_vm_obj_single_datastore",
+    [
+        ["nfs", "nfs", [rhel7_minimal, rhel69_template, ubuntu16_template]],
+    ],
+    indirect=True,
+)
+def test_migration_throttling(
+    request, appliance, mapping_data_multiple_vm_obj_single_datastore
+):
+    """
+    Polarion:
+        assignee: sshveta
+        casecomponent: V2V
+        initialEstimate: 1/4h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+    """
+    migration_settings(appliance).migration_throttling.set_max_migration_per_conv_host("2")
+
+    infrastructure_mapping_collection = appliance.collections.v2v_infra_mappings
+    mapping_data = mapping_data_multiple_vm_obj_single_datastore.infra_mapping_data
+    mapping = infrastructure_mapping_collection.create(**mapping_data)
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
+
+    migration_plan_collection = appliance.collections.v2v_migration_plans
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()),
+        description="desc_{}".format(fauxfactory.gen_alphanumeric()),
+        infra_map=mapping.name,
+        vm_list=mapping_data_multiple_vm_obj_single_datastore.vm_list,
+    )
+    assert migration_plan.wait_for_state("Started")
+    request_details_list = migration_plan.get_plan_vm_list(wait_for_migration=False)
+    vms = request_details_list.read()
+    conversion_host_popup = []
+    # Check if conversion host is shown for each VM
+    for vm in vms:
+        try:
+            popup_text = request_details_list.read_additional_info_popup(vm)
+            # open__additional_info_popup function also closes opened popup in our case
+            request_details_list.open_additional_info_popup(vm)
+            conversion_host_popup.append(popup_text['Conversion Host'])
+        except NoSuchElementException:
+            # conversion host might be empty for throttled vm
+            # until it is waiting for other VM's to finish
```

Comment:
I was thinking it would be probably a good idea to make sure we have at least 2 conv hosts configured and then wait until we see conv host for each vm in its popup. 
That way, we don't assume that `conversion host might be empty for throttled vm` and we actually have conv host for the the vms before we go to assert statement at the end of the test.
what do you think?

Reply:
@kedark3 : All the conversion hosts are configured . There is no control on how and which conversion host is picked .so for a test all three migrations can have two different conv hosts or it might wait for two conversions to finish (in which case it will be empty ) and then start third migration.

Reply:
@sshveta thanks. My assumption was that if we have 2 conversion hosts with 2 migrations limit for each host, in that case for this test with three vms, if we wait until all vms are assigned a conversion host, we will see 2 vms assigned to host1 and 1 vm assigned to host2. That makes it more accurate to verify. See what I am saying? Would that be possible to implement or be too unpredictable?

Reply:
So this sets limit for concurrent migration per conversion host .
Same host can also be picked for all the VM's but only two will run at a given time and third one will start only when one VM is done (if the same host is picked)

Reply:
I get that, but isn't there also supposed to be a feature that distributes the VM Migration across the available hosts evenly? I am almost sure we had that too.  Not trying to drag it too much, just see if we can improve this in any way. Thanks for cooperating.

Reply:
I am not aware of that feature @kedark3 , may be @Yadnyawalkya knows . Yadnya , thoughts ?

Reply:
@kedark3  we do have such RFE (I searched, did not got that link but I am aware there is such RFE) but we don't have ability to choose conversion host per migration atm as @sshveta  mentioned.

Reply:
Added related BZ in comment